hwdb.d/60-keyboard.hwdb

```diff
@@ -186,7 +186,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredator*PH*315-52:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnTravelMateB311-31*:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnTravelMateB311R-31*:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnTravelMateB311RN-31*:pvr*
- KEYBOARD_KEY_8a=f20                                    # Microphone mute
+ KEYBOARD_KEY_8a=micmute                                # Microphone mute
```

Comment:
FTR, these comments are now obsolete. Doesn't hurt, but they were only meant to describe their actual meaning (as "f20" isn't self-descriptive).